include/rocksdb/env.h

```diff
@@ -999,7 +999,7 @@ class WritableFile {
   /*
    * Get the size of valid data in the file.
    */
-  virtual uint64_t GetFileSize() { return 0; }
+  virtual uint64_t GetFileSize() = 0;
```

Comment:
We need to leave the default implementation in place. Otherwise it breaks code compatibility.

Reply:
Thank you for catching this. Do you mean that applications implementing their own subclass of WritableFile will have compilation errors with this change?

Reply:
Yes. We generally try to defer such changes to a major release.

Reply:
Thank you for calling this out. I'm adding this back in #11927